### ISSUE TITLE
Despacito

### ISSUE THREAD

**Issue by hlung** [kelseyhightower/nocode#1650]

**Comment by JerryChin** [comment]
Yes, it even supports 100% *forward compatibility*, feel free to use it!

**Comment by moyoteg** [comment]
Thanks to the The Flying Spaghetti Monster it is backwards compatible!! I want to donate some $LTC or a beer.
